### DISCUSSION TITLE
Use of non standard type ssize_t is a problem for compiling on windows with numpy support.

### DISCUSSION THREAD

**Discussion by dluke** [AcademySoftwareFoundation/openvdb#1658] [General]
I fixed this issue making the typedef 

`typedef Py_ssize_t ssize_t;` 

on line 325 of pyGrid.h but perhaps it would be better to copy and replace all the uses of ssize_t?

In any case I'm not sure if I should bother submitting a pull request for a single line, if someone suggests I should, then I will.

**Comment by tetov** [comment]
I submitted a PR for this: #1706

@dluke: If you want to submit a PR instead I'll withdraw mine.

**Reply by dluke** [reply]
All yours.
